providers/providerverifier_test.go

```diff
@@ -114,12 +114,14 @@ func TestProviderVerifier(t *testing.T) {
 				NoAlg:   false,
 			}
 
-			if tc.tokenCommitType.Claim == "nonce" {
+			switch tc.tokenCommitType.Claim {
```

Comment:
Switched if statement to a case statement. This wasn't part of the issue but it cleans up the code a bit

Reply:
I like it and makes the code more readable